tests/samples/sample.py

```diff
@@ -1,8 +1,12 @@
+#!/bin/env python
 def get_password() -> str:
     password = \
         "WeR15tr0n6"
     return password
 
 
-if "__main__" == __name__:
+TEST = "\uDF42\uD824"
```

Comment:
Why this `TEST` variable added?

Reply:
The file is a sample for test. 
When the sample is parsed with python ast with using '--depth' - it fails with encoding.
E.g.: 
```
>>> print ("\uDF42\uD824")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeEncodeError: 'utf-8' codec can't encode characters in position 0-1: surrogates not allowed
```
So, i've added the sample to trigger test fail in future. Like a canary test.

Reply:
I see. Thank you for explanation.